tests/integration/test_config.py

```diff
@@ -97,8 +93,18 @@ def _config(value):
             capture_output=True,
             text=True,
         )
-        user_agent = [v for v in proc.stderr.splitlines() if "User-Agent" in v]
-        user_agent = user_agent[0].split(":", 1)[-1].strip() if user_agent else ""
+        user_agent = ""
+        for v in proc.stderr.splitlines():
+            # Unfortunately conda has evolved how it logs request headers
+            # So this regular expression attempts to match multiple forms
+            # > User-Agent: conda/...
+            # .... {'User-Agent': 'conda/...', ...}
```

Comment:
Reminder to myself that this sounds like a plugin opportunity (been thinking about a general purpose logging plugin hook, to configure an own handler)